ckanext/canada/templates/admin/index.html

```diff
@@ -1,21 +1,17 @@
 {% ckan_extends %}
 
 {% block primary_content_inner %}
-  {% if h.is_registry() %}
-    {{ super() }}
-  {% endif %}
+  {{ super() }}
 {% endblock %}
```

Comment:
unconditionally calling `{{ super() }}` means this whole block can be removed instead.